calico_node/startup/startup.go

```diff
@@ -771,6 +771,11 @@ func ensureDefaultConfig(c *client.Client, node *api.Node) error {
 		return err
 	}
 
+	// Update the ClusterType with the type in CLUSTER_TYPE
```

Comment:
I think we want plural here - `CLUSTER_TYPES` which could be a comma-separated list of types.

e.g.

```
kops,k8s,aws,kdd,policy-only
```